### DISCUSSION TITLE
Ag1000G phase 3 public dataset

### DISCUSSION THREAD

**Discussion by alimanfoo** [sgkit-dev/sgkit#468] [Show and tell]
Hi all,

I just wanted to share that we've released a new public dataset of genome variation data from the Anopheles gambiae 1000 Genomes Project (Ag1000G). The data include genome-wide SNP calls in 2,784 wild-caught mosquitoes, plus some crosses. Documentation about the data is [available here](https://malariagen.github.io/vector-data/ag3/intro.html).

As part of the data release I've created a Python package to make accessing the data easier, particularly accessing the zarr data in the cloud. The data are not in the native sgkit format, but it is straightforward to create an xarray dataset, and I've added a function to do this, [example here](https://nbviewer.jupyter.org/gist/alimanfoo/deb60020d175cf1079891c9007ad1368). I haven't documented this publicly yet as thought I'd share here and dataset looks right, and feedback welcome.

Hoping this will be a good opportunity to try out sgkit with these new data, there are lots of analyses we'd like to do.

Cheers,
Alistair

**Comment by hammer** [comment]
Thanks @alimanfoo and congratulations on the milestone!

> The data are not in the native sgkit format

Why not? Is there anything we could do to make it more likely for you to choose our format in the future?

**Reply by alimanfoo** [reply]
> Why not? Is there anything we could do to make it more likely for you to choose our format in the future?

It's just a matter of timing. All of the data production work on this predates sgkit. At some point I'm sure we'd be happy to migrate our production pipelines to the new format.
